lib/screens/contact_us_screen.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class ContactUsScreen extends StatelessWidget {
+  const ContactUsScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context)
+        .textTheme
+        .apply(displayColor: Theme.of(context).colorScheme.onSurface);
+    return Expanded(
+      child: ListView(
+        children: <Widget>[
+          const SizedBox(height: 7),
+          TextStyleExample(name: "Contact Us", style: textTheme.displayMedium!),
+          TextStyleExample(
+              name:
+                  "Want to get in touch with Galactic Coders? We would love to hear from you! You can contact us through our website's contact form, or email us at [email protected] You can also follow us on social media to stay up-to-date with our latest projects and events. If you are interested in joining our group or collaborating with us on a project, please don't hesitate to reach out! ",
```

Comment:
If you'd like to try out internationalization the flutter way I recommend using a free account at Localizely https://app.localizely.com/ 

Then you follow the steps in the flutter docs here: https://docs.flutter.dev/development/accessibility-and-localization/internationalization